build/params_wallaby.go

```diff
@@ -17,6 +17,7 @@ import (
 
 var NetworkBundle = "wallaby"
 var BundleOverrides map[actors.Version]string
+var ActorDebugging = true
```

Comment:
Should probably call out that this is changing from true to false.

Reply:
Worth indicating in the handoff to Wallaby that we're enabling this feature now so that node operators expect the extra output in logs?